static/app/gettingStartedDocs/powershell/powershell.tsx

```diff
@@ -0,0 +1,197 @@
+} from 'sentry/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding';
+import {t, tct} from 'sentry/locale';
+import {getPackageVersion} from 'sentry/utils/gettingStartedDocs/getPackageVersion';
+
+type Params = DocsParams;
+
+const getConfigureSnippet = (params: Params) => `
+# You need to import the module once in a script.
+Import-Module Sentry
+
+# Start the Sentry SDK with the default options.
+# It may be helpful when investigating issues with your setup to pass \`-Debug\` to \`Start-Sentry\`.
+# This enables debug logging (\`Write-Debug\`) for the Sentry client.
+# We enable it here for demonstration purposes when first trying Sentry.
+# You shouldn't do this in your applications unless you're troubleshooting issues with Sentry.
+Start-Sentry -Debug {
+    # A Sentry Data Source Name (DSN) is required.
+    # See https://docs.sentry.io/product/sentry-basics/dsn-explainer/
+    # You can set it in the SENTRY_DSN environment variable, or you can set it in code here.
+    $_.Dsn = '${params.dsn}'
+
+    # This option will enable Sentry's tracing features. You still need to start transactions and spans.
+    # For example, setting the rate to 0.1 would capture 10% of transactions.
+    $_.TracesSampleRate = 1.0
+}
+
+# Later on in your production script, you should omit the \`-Debug\` flag.:
+Start-Sentry {
+    $_.Dsn = '${params.dsn}'
+    $_.TracesSampleRate = 0.1
+}`;
+
+const getPerformanceMonitoringSnippet = () => `
+# Transaction can be started by providing, at minimum, the name and the operation
+$transaction = Start-SentryTransaction 'test-transaction-name' 'test-transaction-operation'
+
+# Transactions can have child spans (and those spans can have child spans as well)
+$span = $transaction.StartChild("test-child-operation")
+# ...
+# (Perform the operation represented by the span/transaction)
+# ...
+$span.Finish() # Mark the span as finished
+
+$span = $transaction.StartChild("another-span")
+# ...
+$span.Finish()
+
+$transaction.Finish() # Mark the transaction as finished and send it to Sentry`;
+
+const onboarding: OnboardingConfig = {
+  introduction: () =>
+    t(
+      'Sentry for PowerShell module supports PowerShell 7.2+ on Windows, macOS, and Linux as well as Windows PowerShell 5.1+.'
+    ),
+  install: params => [
+    {
+      type: StepType.INSTALL,
+      description: t('Install the module:'),
+      configurations: [
+        {
+          partialLoading: params.sourcePackageRegistries.isLoading,
+          code: [
+            {
+              language: 'powershell',
+              label: 'Install Module',
+              value: 'powershellget',
+              code: `Install-Module -Name Sentry -Repository PSGallery -RequiredVersion ${getPackageVersion(params, 'sentry.powershell', '1.0.0')} -Force`,
+            },
+          ],
+        },
+      ],
+    },
+  ],
+  configure: params => [
+    {
+      type: StepType.CONFIGURE,
+      description: tct('Initialize the SDK as early as possible.'),
```

Comment:
That is the typescript error, the tests run into:
```suggestion
      description: t('Initialize the SDK as early as possible.'),
```